Translations update from Mattermost Weblate

weblate opened the pull request:
Translations update from [Mattermost Weblate](https://translate.mattermost.com) for [Calls/server](https://translate.mattermost.com/projects/calls/server/).



Current translation status:

![Weblate translation status](https://translate.mattermost.com/widget/calls/server/horizontal-auto.svg)

Commit:
Translated using Weblate (Persian)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/fa/

Commit:
Translated using Weblate (Italian)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/it/

Commit:
Translated using Weblate (Italian)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/it/

Commit:
Translated using Weblate (Korean)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/ko/

Commit:
Translated using Weblate (Korean)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/ko/

Commit:
Translated using Weblate (Lithuanian)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/lt/

Commit:
Translated using Weblate (Lithuanian)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/lt/

Commit:
Translated using Weblate (Croatian)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/hr/

Commit:
Translated using Weblate (Arabic)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/ar/

Commit:
Translated using Weblate (Spanish)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/es/

Commit:
Translated using Weblate (Turkish)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/tr/

Commit:
Translated using Weblate (Turkish)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/tr/

Commit:
Translated using Weblate (Vietnamese)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/vi/

Commit:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/zh_Hant/

Commit:
Translated using Weblate (Italian)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/it/

Commit:
Translated using Weblate (Korean)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/ko/

Commit:
Translated using Weblate (Lithuanian)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/lt/

Commit:
Translated using Weblate (Spanish)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/es/

Commit:
Translated using Weblate (Turkish)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/tr/

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

ctlaltdieliet reviewed (commented):
lgtm

ctlaltdieliet reviewed (approved):
lgtm

ctlaltdieliet reviewed (approved):
still lgtm

Commit:
Translated using Weblate (Korean)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/ko/

ctlaltdieliet reviewed (approved):
lgtm :rocket:

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

streamer45 commented:
@ctlaltdieliet e2e tests pipeline is half broken for some reason. I'll be investigating tomorrow (tracked via https://mattermost.atlassian.net/browse/MM-64474).

ctlaltdieliet commented:
Thank you @streamer45 
Please, feel no pressure. There are a lot of other strings to translate in the Mattermost project!

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

Commit:
Translated using Weblate (Arabic)

Currently translated at 21.4% (3 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/ar/

Commit:
Translated using Weblate (Spanish)

Currently translated at 7.1% (1 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/es/

Commit:
Translated using Weblate (Lithuanian)

Currently translated at 7.1% (1 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/lt/

Commit:
Translated using Weblate (Turkish)

Currently translated at 7.1% (1 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/tr/

Commit:
Translated using Weblate (Korean)

Currently translated at 0.0% (0 of 14 strings)

Translation: Calls/server
Translate-URL: https://translate.mattermost.com/projects/calls/server/ko/
